test/unit/acc/src/AccTagTest.cpp

```diff
@@ -57,13 +57,13 @@ TEMPLATE_LIST_TEST_CASE("test all possible acc tag combinations", "[acc][tag]",
     {
         if constexpr(std::is_same_v<TestTag, ExpectedTag>)
         {
-            STATIC_REQUIRE((std::is_same_v<alpaka::trait::AccToTagType<TestAcc>, ExpectedTag>) );
-            STATIC_REQUIRE((std::is_same_v<alpaka::trait::TagToAccType<TestTag, Dim, Idx>, TestAcc>) );
+            STATIC_REQUIRE((std::is_same_v<alpaka::trait::Tag<TestAcc>, ExpectedTag>) );
+            STATIC_REQUIRE((std::is_same_v<alpaka::trait::Acc<TestTag, Dim, Idx>, TestAcc>) );
```

Comment:
This is how I would like the API to look like:
```suggestion
            STATIC_REQUIRE((std::is_same_v<alpaka::Tag<TestAcc>, ExpectedTag>) );
            STATIC_REQUIRE((std::is_same_v<alpaka::Acc<TestTag, Dim, Idx>, TestAcc>) );
```